packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -370,6 +389,37 @@ class AndroidCameraCameraX extends CameraPlatform {
     return _cameraEvents(cameraId).whereType<VideoRecordedEvent>();
   }
 
+  /// Locks the capture orientation.
+  @override
+  Future<void> lockCaptureOrientation(
+    int cameraId,
+    DeviceOrientation orientation,
+  ) async {
+    // Flag that (1) default rotation for UseCases will need to be set manually
+    // if orientation is ever unlocked and (2) the capture orientation is locked
+    // and should not be changed until unlocked.
+    if (!shouldSetDefaultRotation) {
+      shouldSetDefaultRotation = true;
+    }
+    captureOrientationLocked = true;
+
+    // Get target rotation based on locked orientation.
+    final int targetLockedRotation =
+        _getRotationConstantFromDeviceOrientation(orientation);
+
+    // Update UseCases to use target device orientation.
```

Comment:
The Preview usecase also has a method to set the target rotation, should we be setting that as well? 
https://developer.android.com/reference/kotlin/androidx/camera/core/Preview#setTargetRotation(int)

Reply:
I don't believe so since this is only supposed to impact the capture orientation. The preview rotation should be managed by the Flutter app using the plugin like, for example, [our example app](https://github.com/flutter/packages/blob/main/packages/camera/camera_android_camerax/example/lib/camera_preview.dart).